spanner/google/cloud/spanner/database.py

```diff
@@ -179,7 +180,8 @@ def ddl_statements(self):
     def spanner_api(self):
         """Helper for session-related API calls."""
         if self._spanner_api is None:
-            self._spanner_api = SpannerClient()
+            self._spanner_api = SpannerClient(lib_name='gccl',
+                                              lib_version=__version__)
```

Comment:
@jonparrott Has convinced me of the merits of "one continuation, all continuation", so here

```python
            self._spanner_api = SpannerClient(
                lib_name='gccl', lib_version=__version__)
```